plugins/dev9ghzdrk/Win32/Win32.cpp

```diff
@@ -77,7 +77,7 @@ void OnOk(HWND hW) {
 	if (i == -1)
 	{
 		//adapter not selected
-		if (Button_GetCheck(GetDlgItem(hW, IDC_ETHENABLED)))
+		if ( config.ethEnable = Button_GetCheck(GetDlgItem(hW, IDC_ETHENABLED)))
```

Comment:
This if statement will only run if no adapter is selected, As this is now where the config value is saved, you won't be able to enable, or disable the pulgin's ethernet emulation if you've selected an adapter.

Reply:
Good point. If you can squash the change in (revert back to two separate calls), great. If not I'll do it in a few hours when I'm able